kafka/metadata.csv

```diff
@@ -8,8 +8,8 @@ kafka.request.channel.queue.size,gauge,10,request,,Number of queued requests.,-1
 kafka.request.fetch.failed,gauge,10,request,,Number of client fetch request failures.,-1,kafka,req fetch fail
 kafka.request.fetch.failed_per_second,gauge,10,request,second,Rate of client fetch request failures per second.,-1,kafka,req fetch fail rate
 kafka.request.handler.avg.idle.pct,gauge,10,fraction,,Average fraction of time the request handler threads are idle.,1,kafka,req handle avg idle
-kafka.request.produce.time.avg,gauge,10,request,second,Average time for a produce request.,0,kafka,req prod time avg
-kafka.request.produce.time.99percentile,gauge,10,request,second,Time for produce requests for 99th percentile.,0,kafka,req prod time 99
+kafka.request.produce.time.avg,gauge,10,rate,millisecond,Average time for a produce request.,0,kafka,req prod time avg
```

Comment:
This instead should be
```suggestion
kafka.request.produce.time.avg,gauge,10,millisecond,,Average time for a produce request.,0,kafka,req prod time avg
```
`millisecond` is the `unit_name` and because the metric is not a ratio, `per_unit_name` should stay empty